src/components/autocomplete/demoRepeatMode/index.html

```diff
@@ -0,0 +1,83 @@
+    <form ng-submit="$event.preventDefault()">
+      <p id="autocompleteDescription">
+        Use the <code>md-mode</code> attribute to fall back to <code>ng-repeat</code>
+        for autocomplete entries. Note that sizable lists can have performance
+        implications for your application. Using 'standard' mode will require
+        you to address those directly within your application.
+      </p>
+      <div class="autocomplete-instance-wrapper">
+        <h4>Standard mode</h4>
+        <md-autocomplete
+            id="custom-template"
+            ng-disabled="ctrl.isDisabled"
+            md-no-cache="ctrl.noCache"
+            md-selected-item="ctrl.selectedItem"
+            md-search-text-change="ctrl.searchTextChange(ctrl.searchText)"
+            md-search-text="ctrl.searchText"
+            md-selected-item-change="ctrl.selectedItemChange(item)"
+            md-items="item in ctrl.querySearch(ctrl.searchText)"
+            md-item-text="item.name"
+            md-min-length="0"
+            input-aria-label="Current Repository"
+            placeholder="Pick an Angular repository"
+            md-menu-class="autocomplete-custom-template"
+            md-menu-container-class="custom-container"
+            md-mode="standard">
+          <md-item-template>
+            <span class="item-title">
+              <md-icon md-svg-icon="img/icons/octicon-repo.svg"></md-icon>
+              <span> {{item.name}} </span>
+            </span>
+            <p class="item-desc">{{item.desc}}</p>
+            <span class="item-metadata">
+              <span>
+                <strong>{{item.watchers}}</strong> watchers
+              </span>
+              <span>
+                <strong>{{item.forks}}</strong> forks
+              </span>
+            </span>
+          </md-item-template>
+        </md-autocomplete>
+      </div>
+
+      <div class="autocomplete-instance-wrapper">
+        <h4>Virtual mode</h4>
+        <md-autocomplete
+            id="custom-template"
+            ng-disabled="ctrl.isDisabled"
+            md-no-cache="ctrl.noCache"
+            md-selected-item="ctrl.selectedItem"
+            md-search-text-change="ctrl.searchTextChange(ctrl.searchText)"
+            md-search-text="ctrl.searchText"
+            md-selected-item-change="ctrl.selectedItemChange(item)"
+            md-items="item in ctrl.querySearch(ctrl.searchText)"
+            md-item-text="item.name"
+            md-min-length="0"
+            input-aria-label="Current Repository"
+            placeholder="Pick an Angular repository"
+            md-menu-class="autocomplete-custom-template"
+            md-menu-container-class="custom-container"
+            md-mode="virtual">
+          <md-item-template>
+            <span class="item-title">
+              <md-icon md-svg-icon="img/icons/octicon-repo.svg"></md-icon>
+              <span> {{item.name}} </span>
+            </span>
+            <p class="item-desc">{{item.desc}}</p>
```

Comment:
For virtual mode, this needs to stay on one line, so we need to do something like this:
```html
<p class="item-desc">{{item.desc | limitTo:40}}{{item.desc.length > 40 ? '...': ''}}</p>
```

Reply:
Ah. Sounds good :+1: